README.md

```diff
@@ -67,12 +67,12 @@ Asserts that the target is False:
 expect(False).to.be.false()
 ```
 
-#### instanceof
+#### instance_of
```

Comment:
Yeah @catriuspham definitely should revert this change. It's breaking backwards compatibility for no reason. 

`instanceof` is also commonly written that way. Like the `sizeof` operator. They are treated exceptionally.